src/main/scala/org/broadinstitute/dsde/firecloud/core/ProfileClient.scala

```diff
@@ -340,14 +340,15 @@ class ProfileClientActor(requestContext: RequestContext) extends Actor with Fire
       case None => Put
     }
 
-    memberList flatMap { members => pipeline { rawlsRequestMethod(url, members) } }
-
-    // don't leak any sensitive data
-    RequestComplete(NoContent)
-  }.recover {
+    memberList flatMap { members => pipeline { rawlsRequestMethod(url, members) } } map { response =>
+      if(response.status.isFailure)
+        throw new Exception("Error synchronizing NIH whitelist")
```

Comment:
If we throw any kind of error here, we'll be percolating a non-fatal exception back down to the user. That error will be useless to the user. I think we should log it and catch it as flagged alert.